docs/content/en/docs/Upgrade/manual.md

```diff
@@ -44,3 +44,18 @@ To specify a specific image, use the `--image` flag:
 ```bash
 sudo kairos-agent upgrade --image <image>
 ```
+
+
+To upgrade with a docker image behind a registry with authentication, the upgrade command provides the following flags:
+
+ - --auth-username
+ - --auth-password
+ - --auth-server-address
+ - --auth-type
+ - --auth-registry-token
+ - --auth-identity-token
```

Comment:
Unfortunately, hugo converts two dashes into one long dash. To avoid confusion, I'd put each of these between back ticks

![image](https://user-images.githubusercontent.com/433958/232771583-216f9fca-b082-4e5b-9a95-4ee498777c65.png)

Reply:
I think also tables display way better, along with the description of what each flag does

Reply:
Good point. Will change it. Also, I need to understand what those options mean, as I have only used the user and the password lol